pkg/scheduler/framework/plugins/volumebinding/volume_binding_test.go

```diff
@@ -660,7 +661,11 @@ func TestVolumeBinding(t *testing.T) {
 				}
 			}
 
-			pl, err := New(args, fh)
+			fts := feature.Features{
+				EnableVolumeCapacityPriority: utilfeature.DefaultFeatureGate.Enabled(features.VolumeCapacityPriority),
```

Comment:
instead, change the test cases to have a feature.Features struct. If we are decoupling, there is no reason for the unit tests to depend on the feature gates.

Reply:
fixed in https://github.com/kubernetes/kubernetes/pull/103493/commits/1a012fc11f767848af8bb7aa0bb735b4a622f4b1
because the validation code depends on feature gates, I need to add a new function which we can pass `feature.Features` struct